### PR TITLE
Remove unused variables

### PR DESCRIPTION
**Pull request by practicalswift**

### CODE REVIEW

**Comment by fanquake** [comment]
Can you please rename the commits to remove the [trivial], as that tag is really only used when a PR doesn't touch any code.

**Comment by jtimon** [comment]
I have used "Trivial: " in the beginning of the message of some commits that changed code in trivial ways, I didn't know that policy. When I don't touch code at all I tend to just use "Doc: ", but I did these things on my own, if we have a place describing these kind of best practices I missed it, can you link to it?

**Comment by jtimon** [comment]
utACK 180df23

File: src/addrman.cpp [hunk 1]
```diff
@@ -54,11 +54,8 @@ double CAddrInfo::GetChance(int64_t nNow) const
 {
     double fChance = 1.0;
 
-    int64_t nSinceLastSeen = nNow - nTime;
     int64_t nSinceLastTry = nNow - nLastTry;
```
Comment (robmcl4) [inline src/addrman.cpp]: Maybe initializing `nSinceLastTry` and the following `if` statement can be collapsed into a call to `std::max()`.

**Review by dcousens — APPROVED** [review]

**Comment by paveljanik** [comment]
ACK https://github.com/bitcoin/bitcoin/pull/9532/commits/180df23e2006e075f46a1048f29598a3c44f93b7

File: src/addrman.cpp (hunk 1 above)
Comment (practicalswift) [inline src/addrman.cpp]: Good idea! I'll take that in another PR!

**Comment by practicalswift** [comment]
@fanquake Removed `[trivial]` from the commit titles as requested :-)

**Comment by instagibbs** [comment]
@jtimon https://github.com/bitcoin/bitcoin/pull/9542 PR about [trivial] tag

File: src/addrman.cpp (hunk 1 above)
Comment (practicalswift) [inline src/addrman.cpp]: @robmcl4 Submitted as #9553 :-)

**Comment by practicalswift** [comment]
I believe this one should be ready to merge. Please let me know if any adjustments are needed and I'll fix it right away! :-)

**Comment by jtimon** [comment]
utACK 90fd29b (I didn't made sure all the variables are unused manually again for a full re-ACK)

**Comment by practicalswift** [comment]
Any sponsor who has time to take a look at this hopefully trivial PR? :-)

**Comment by MarcoFalke** [comment]
utACK 90fd29bd0dcbaf5d5c4ef9e69824297c35f0b88e

**Pull request merged**